Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 gem 'github-pages'
+<<<<<<< HEAD
+gem 'jekyll-paginate'
```

Comment:
I don't think that you need to add this directly as it's pulled in by the github-pages gem